### DISCUSSION TITLE
Api

### DISCUSSION THREAD

**Discussion by 243691510** [consenlabs/webview#46] [Q&A] [answered]
我們想用你的錢包，做我們app收轉帳服務，但是我們在對接時候不知道在那裡獲得參數
![FB3D1E58-E691-4289-906D-130822344D32](https://user-images.githubusercontent.com/8498297/167988239-df172eaa-18ae-43cb-8727-0e2fbb1506d5.png)

**Comment by izayl** [comment, marked as answer]
如果用的是 `ethers.js` 的話，建議參考 ethers 的文檔：https://docs.ethers.io/v5/api/providers/types/#providers-TransactionRequest 了解下

**Reply by 243691510** [reply]
我们用的uniapp+php ，前端uniapp，后端php.有没有例子或者要引什么sdk .可以实现

**Reply by izayl** [reply]
你截圖裡的 `tx` 的類型就是這裡定義的 [TransactionRequest](https://docs.ethers.io/v5/api/providers/types/#providers-TransactionRequest)

**Reply by izayl** [reply]
> 我们用的uniapp+php ，前端uniapp，后端php.有没有例子或者要引什么sdk .可以实现

可以參考這個項目 https://github.com/scaffold-eth/scaffold-eth
